src/Compilers/CSharp/Portable/FlowAnalysis/FlowAnalysisPass.cs

```diff
@@ -78,6 +80,25 @@ internal class FlowAnalysisPass
             return block;
         }
 
+        private static BoundBlock AppendImplicitReturn(BoundBlock body, bool originalBodyNested, MethodSymbol method, CSharpSyntaxNode syntax)
```

Comment:
I might have put the new flag at the end to make it easier to use named arguments.